tests/roles/barbican_adoption/tasks/main.yaml

```diff
@@ -0,0 +1,54 @@
+- name: deploy podified Barbican
```

Comment:
Adding here -- need to look at keystone_adoption.  There is a task there to remove old endpoints.  You need to remove the barbican endpoint too.

Reply:
ditto for stop_openstack_services